### DISCUSSION TITLE
when i enable "ChromeRuntime", the ChromiumWebBrowser layouts incorrect

### DISCUSSION THREAD

**Discussion by ijustkeepmoving** [cefsharp/CefSharp#4778] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

120.2.50

### Operating System

Windows 10

### Architecture

x64

### .Net Version

CefSharp.Wpf.NETCore

### Implementation

WPF

### Reproduction Steps

hi~~ i am a newer to Cefsharp~~~  

when i enable  "settings.ChromeRuntime = true;" the  position  and size of  browser is not working, it layouts on the top and left of the window with a fixed size~~~

when i set settings.ChromeRuntime = false; everything goes well

i have test the MinmalExample ,the result is the same with description  above

i dont know why?
<img width="932" alt="微信图片_20240405095629" src="https://github.com/cefsharp/CefSharp/assets/12046942/525b60bf-ba55-4853-af87-7322bcfaac6f">

### Expected behavior

 the position and size  render  correct~~

### Actual behavior

 the position and size  is not working

### Does this problem also occur in the CEF Sample Application

Yes using WPF/OffScreen command line args

**Comment by ijustkeepmoving** [comment]
can anyone help me?

**Reply by amaitland** [reply]
This is a known limitation of the `Chrome` runtime.

> The WPF and OffScreen versions cannot use the Chrome Runtime yet, see https://github.com/chromiumembedded/cef/issues/3293

As per https://github.com/cefsharp/CefSharp/discussions/4123

You should be able to use https://github.com/cefsharp/CefSharp.Wpf.HwndHost

**Reply by ijustkeepmoving** [reply]
THANKS  A LOT

**Reply by ijustkeepmoving** [reply]
It seems that the sock5 proxy cannot be used with [CefSharp.Wpf.HwndHost](https://github.com/cefsharp/CefSharp.Wpf.HwndHost),

pages can not be loaded when i set a proxy to it;

 the code is :
  v["mode"] = "fixed_servers";
  v["server"] = $"socks5://xxxxxx";
  string error = string.Empty;
  bool ret = requestContext.SetPreference("proxy", v, out error);

but the proxy works with CefSharp.Wpf.NETCore~~~

**Reply by ijustkeepmoving** [reply]
the proxy is socks5 with IPV6 address

**Reply by ijustkeepmoving** [reply]
OK, it's my problem,thanks~~~
